src/Routing/RouteBuilder.php

```diff
@@ -519,6 +521,35 @@ protected function _methodRoute($method, $template, $target, $name)
     }
 
     /**
+     * Load routes from a plugin.
+     *
+     * The routes file will have a local variable named `$routes` made available which contains
```

Comment:
I was trying to say the routes file will have access to `$routes` as a local variable.